FTheoryToolsInOSCAR.ipynb

```diff
@@ -2,24 +2,24 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "id": "a4518a50-2354-4d56-8315-d6edc7a4b0f9",
+   "id": "85fd2ee3-b210-48cb-ae68-e8d254dd7703",
    "metadata": {},
    "source": [
     "# *FTheoryTools*\n",
     "\n",
-    "* Authors: Martin Bies, Andrew P. Turner\n",
-    "* Version: OSCAR version 1.0.0\n",
-    "* Last modified: April 3, 2024"
+    "* Authors: Martin Bies, Andrew P. Turner, Mikelis Emils Mikelsons\n",
```

Comment:
We sort authors alphabetically by their last names. It should thus be Bies, Mikelsons, Turner. Please adjust.